src/dsp-insts.rkt

```diff
@@ -53,10 +59,39 @@
              (length v-acc))
           "VECTOR-MAC: length of vectors not equal")
   (for/list ([e-acc v-acc]
-               [e1 v1]
-               [e2 v2])
+             [e1 v1]
+             [e2 v2])
     (box (bvadd (unbox e-acc) (bvmul (unbox e1) (unbox e2))))))
 
+; Define sine and cosine as an interpreted functions
+(define-symbolic cosine (~> (bitvector (value-fin))
```

Comment:
Should we consider automatically generating a symbolic function when we hit and unknown operations instead of having hard coded ones?